README.md

```diff
@@ -99,9 +99,11 @@ The "Sanity Test" is a test which runs all checks against all the test sources f
 
 The "Plugin Test" is an integration test suite which verifies plugin features such as metric calculation, coverage etc. To launch it:
 
-    mvn clean install -Pit-plugin
+    mvn clean install -Pit-plugin -DskipTests=false -DcommunityEditionTestsOnly=true
```

Comment:
It should work with `-Pit-plugin` and without `-DskipTests=false` because it's what the profile `it-plugin` is supposed to do in `its/plugin/pom.xml`:
```xml
    <profile>
      <id>it-plugin</id>
      <properties>
        <skipTests>false</skipTests>
      </properties>
    </profile>
```